pages/profiles/[publicKey]/offers.tsx

```diff
@@ -117,154 +122,162 @@ const OfferPage: NextPage<WalletDependantPageProps> = ({ publicKey, ...props })
               const defaultListing = receivedOffer?.listings.find(
                 (listing) => listing.auctionHouse.toString() === HOLAPLEX_MARKETPLACE_ADDRESS
               );
-              return receivedOffer.offers?.map((offer) => (
-                <div
-                  key={offer.address}
-                  className={`flex h-28 flex-row justify-between rounded-lg border border-gray-800 p-4`}
-                >
-                  <div className={`flex items-center justify-start`}>
-                    <img
-                      src={imgOpt(receivedOffer?.image, 800)!}
-                      className={`h-20 rounded-lg`}
-                      alt={receivedOffer?.name}
-                    />
-                    <div>
-                      <div className={`ml-4 flex flex-col justify-center`}>
-                        <p className={`mb-0 text-base text-gray-300`}>
-                          <Link href={`/profiles/${offer.buyer}/offers`}>
-                            <a className={`text-white hover:text-gray-300`}>
-                              {offer.buyer === userPK?.toBase58()
-                                ? `You`
-                                : `@${shortenAddress(offer.buyer)}`}
-                            </a>
-                          </Link>{' '}
-                          offered{' '}
-                          <span className={`text-white`}>
-                            <DisplaySOL amount={offer.price} iconVariant={`small`} />
-                          </span>{' '}
-                          for{' '}
-                          <Link href={`/nfts/${receivedOffer.address}`}>
-                            <a className={`text-white hover:text-gray-300`}>{receivedOffer.name}</a>
-                          </Link>
-                        </p>
-                        <p className={`mb-0 mt-2 text-base text-gray-500`}>
-                          {formatTime(offer.createdAt, `en_US`)}
-                        </p>
+              return receivedOffer.offers
+                ?.sort(byDate)
+                .reverse()
```

Comment:
You can just reverse the sorting function instead of tacking on a reverse() here :)